src/mmu.c

```diff
@@ -25,20 +25,28 @@ static const uint8_t bootrom[256] = {
     0xF5, 0x06, 0x19, 0x78, 0x86, 0x23, 0x05, 0x20, 0xFB, 0x86, 0x20, 0xFE, 0x3E, 0x01, 0xE0, 0x50
 };
 
-ErrorCode mem_init(void) {
-    uint8_t *rom_bytes = get_rom_bytes();
-
+ErrorCode mmu_init(void) {
     mem = calloc(1, 0x10000);
```

Comment:
Why `calloc` here if the ROM size is known at compile time?

Reply:
This is "fixed" in #25.